earthaccess/store.py

```diff
@@ -63,11 +63,11 @@ def __repr__(self) -> str:
 def _open_files(
     url_mapping: Mapping[str, Union[DataGranule, None]],
     fs: fsspec.AbstractFileSystem,
-    threads: Optional[int] = 8,
+    threads: int = 8,
 ) -> List[EarthAccessFile]:
     def multi_thread_open(data: tuple) -> EarthAccessFile:
-        urls, granule = data
```

Comment:
I think `urls` is correct on this one, the relation is a granule can have many files/urls.

Reply:
The tuple is of type `tuple[str, Optional[DataGranule]]`, so the str is a _singular_ url. It comes from the mapping returned by [_get_url_granule_mapping](https://github.com/nsidc/earthaccess/blob/v0.11.0/earthaccess/store.py#L96) (as well as a couple of other places using `Mapping[str, Optional[DataGranule]]`).

Reply:
I refined the type of the `data` argument to reflect this.